### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by macko1**

### CODE REVIEW

**Comment by macko1** [comment]
better readability, format, language etc.

np, review when free, thanks

File: README.md [hunk 1]
```diff
@@ -1,67 +1,89 @@
 # erouska-android
 
-eRouška ("rouška" means "face mask" in Czech) is an app which helps to fight against COVID-19. When turned on, it scans other users of the app via Bluetooth and saves this data locally. When some user tests positive for COVID-19, he/she is asked to upload their data. Then healtcare authorities can see phone numbers of users of the app which were in close contact with the infected and can call them about further measures.
+## [erouska.cz](https://erouska.cz)
```
Comment (davidvavra) [inline README.md:3]: Name of the repo should be the header, it's common practice

File: README.md [hunk 2]
```diff
@@ -1,67 +1,89 @@
...
+## [Download from Google Play](https://play.google.com/store/apps/details?id=cz.covid19cz.erouska)
```
Comment (davidvavra) [inline README.md:4]: Link should not be same header as previous line, it doesn't look good

File: README.md [hunk 3]
```diff
@@ -1,67 +1,89 @@
...
+- The scanning can be turned off manually at any time.
+- The broadcasted _Device ID_ is changed every hour, so a user cannot be tracked with it. (Only our backend has a knowledge of which _Device ID_'s correspond to which phone number.)
+- The data are kept on backend for 6 hours, then deleted.
+- eRouška is developed open-source from day one.
+
+## Who is developing eRouška?
+
+We are an initiative consisting of people from various Czech IT companies and volunteers called [COVID19CZ](https://covid19cz.cz). We are all unpaid volunteers. 
+
+We are on Slack! [covid19cz.slack.com](covid19cz.slack.com), channel _#erouska_.
```
Comment (davidvavra) [inline README.md:32]: We don't want everyone to join Slack, only the biggest contributors. I would remove this and  only explained it in Contributing section.

File: README.md [hunk 4]
```diff
@@ -1,67 +1,89 @@
...
 
-We are happy to accept pull requests. If you want to help more, discuss with us etc., please join [COVID19CZ](https://covid19cz.cz) Slack (channel #erouska), internally we use [Trello](https://trello.com/b/4xN2Eeqv/bug-wf) for bugs.
+Run:
+`./gradlew assembleDevDebug`
+
+## Contributing
+We are happy to accept pull requests! See [Git Workflow](#git-workflow).
 
-## Git flow
+We are on Slack! [covid19cz.slack.com](covid19cz.slack.com), channel _#erouska_.
```
Comment (davidvavra) [inline README.md:73]: Mention that joining Slack is just for most active contributors, not for everyone.

**Comment by davidvavra** [comment]
I used your changes in PR #164 , closing this one.

**Pull request closed without merging**